### ISSUE TITLE
Unnecessary meta data printed out when deploying a Jupyter Notebook with Voila

### ISSUE THREAD

**Issue by arghyadeep99** [voila-dashboards/voila#569]
When I tried to deploy an interactive Jupyter Notebook using Voila, I get the following unwanted metadata on the webpage:
https://www.writeurl.com/text/9bvpz4p09p0t5c2wir7o/z8dlg6ic4wldor2ruxnc

Why is it so? How can it be resolved?

**Comment by vidartf** [comment]
Would you mind pasting the output in a comment? You can use this for large output:
```
<details><summary>Header</summary>
<pre>
paste here
</pre>
</details>
```

It will look like this:
<details><summary>Header</summary>
<pre>
paste here
</pre>
</details>

**Comment by arghyadeep99** [comment]
Sure man, here it is. 
P.S.:But it's too big.
<details><summary>Extra Metadata at end of notebook</summary>
<pre>
\n \n \n
\n \n \n\n"}]}}, "84b49c111c0146dda38f8cad2ba5a874": {"model_module": "@jupyter-widgets/controls", "model_name": "VBoxModel", "state": {"_dom_classes": [], "_model_module": "@jupyter-widgets/controls", "_model_module_version": "1.5.0", "_model_name": "VBoxModel", "_view_count": null, "_view_module": "@jupyter-widgets/controls", "_view_module_version": "1.5.0", "_view_name": "VBoxView", "box_style": "", "children": ["IPY_MODEL_8040d175227e4f688fdfb080a4fcf016"], "layout": "IPY_MODEL_8cdf8fd2064d4e059fb25d9687a5a68c"}}, "87887dd6dff24e37b6fdcd69cf8b9cf3": {"model_module": "@jupyter-widgets/controls", "model_name": "TextModel", "state": {"_dom_classes": [], "_model_module": "@jupyter-widgets/controls", "_model_module_version": "1.5.0", "_model_name": "TextModel", "_view_count": null, "_view_module": "@jupyter-widgets/controls", "_view_module_version": "1.5.0", "_view_name": "TextView", "continuous_update": true, "description": "n", "description_tooltip": null, "disabled": false, "layout": "IPY_MODEL_6ff598d49223485eab1916aa80e3f9bc", "placeholder": "\u200b", "style": "IPY_MODEL_a1917f7582e5416ba8d538690033e4be", "value": "10"}}, "8cdf8fd2064d4e059fb25d9687a5a68c": {"model_module": "@jupyter-widgets/base", "model_name": "LayoutModel", "state": {"_model_module": "@jupyter-widgets/base", "_model_module_version": "1.2.0", "_model_name": "LayoutModel", "_view_count": null, "_view_module": "@jupyter-widgets/base", "_view_module_version": "1.2.0", "_view_name": "LayoutView", "align_content": null, "align_items": null, "align_self": null, "border": "solid 2px green", "bottom": null, "display": "none", "flex": null, "flex_flow": null, "grid_area": null, "grid_auto_columns": null, "grid_auto_flow": null, "grid_auto_rows": null, "grid_column": null, "grid_gap": null, "grid_row": null, "grid_template_areas": null, "grid_template_columns": null, "grid_template_rows": null, "height": null, "justify_content": null, "justify_items": null, "left": null, "margin": null, "max_height": null, "max_width": null, "min_height": null, "min_width": null, "object_fit": null, "object_position": null, "order": null, "overflow": null, "overflow_x": null, "overflow_y": null, "padding": null, "right": null, "top": null, "visibility": null, "width": null}}, "9359b8eee4e64ffba3545780f1d031e6": {"model_module": "@jupyter-widgets/base", "model_name": "LayoutModel", "state": {"_model_module": "@jupyter-widgets/base", "_model_module_version": "1.2.0", "_model_name": "LayoutModel", "_view_count": null, "_view_module": "@jupyter-widgets/base", "_view_module_version": "1.2.0", "_view_name": "LayoutView", "align_content": null, "align_items": null, "align_self": null, "border": null, "bottom": null, "display": null, "flex": null, "flex_flow": null, "grid_area": null, "grid_auto_columns": null, "grid_auto_flow": null, "grid_auto_rows": null, "grid_column": null, "grid_gap": null, "grid_row": null, "grid_template_areas": null, "grid_template_columns": null, "grid_template_rows": null, "height": null, "justify_content": null, "justify_items": null, "left": null, "margin": null, "max_height": null, "max_width": null, "min_height": null, "min_width": null, "object_fit": null, "object_position": null, "order": null, "overflow": null, "overflow_x": null, "overflow_y": null, "padding": null, "right": null, "top": null, "visibility": null, "width": null}}, "951e6e2a6ae640d8be6873397394f6a5": {"model_module": "@jupyter-widgets/output", "model_name": "OutputModel", "state": {"_dom_classes": [], "_model_module": "@jupyter-widgets/output", "_model_module_version": "1.0.0", "_model_name": "OutputModel", "_view_count": null, "_view_module": "@jupyter-widgets/output", "_view_module_version": "1.0.0", "_view_name": "OutputView", "layout": "IPY_MODEL_169a168310c743719ff439e829768bbb", "msg_id": "", "outputs": [{"metadata": {"tags": []}, "output_type": "display_data", "text/html": "\n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n
  | country | last_update | lat | long_ | confirmed | deaths | recovered | active
-- | -- | -- | -- | -- | -- | -- | -- | --
US | 2020-04-05 04:15:45 | 40.000000 | -100.000000 | 312146 | 8499 | 14997 | 0
Spain | 2020-04-05 04:15:23 | 40.463667 | -3.749220 | 126168 | 11947 | 34219 | 80002
Italy | 2020-04-05 04:15:23 | 41.871900 | 12.567400 | 124632 | 15362 | 20996 | 88274
Germany | 2020-04-05 04:15:23 | 51.165700 | 10.451500 | 96092 | 1444 | 26400 | 68248
France | 2020-04-05 04:15:23 | 46.227600 | 2.213700 | 90848 | 7574 | 15572 | 67702
China | 2020-04-05 02:24:59 | 30.592800 | 114.305500 | 82574 | 3333 | 77160 | 2081
Iran | 2020-04-05 04:15:23 | 32.427908 | 53.688046 | 55743 | 3452 | 19736 | 32555
United Kingdom | 2020-04-05 04:15:23 | 55.000000 | -3.000000 | 42479 | 4320 | 215 | 37944
Turkey | 2020-04-05 04:15:23 | 38.963700 | 35.243300 | 23934 | 501 | 786 | 22647
Switzerland | 2020-04-05 04:15:23 | 46.818200 | 8.227500 | 20505 | 666 | 6415 | 13424
, "text/plain": ""}]}}, "a1917f7582e5416ba8d538690033e4be": {"model_module": "@jupyter-widgets/controls", "model_name": "DescriptionStyleModel", "state": {"_model_module": "@jupyter-widgets/controls", "_model_module_version": "1.5.0", "_model_name": "DescriptionStyleModel", "_view_count": null, "_view_module": "@jupyter-widgets/base", "_view_module_version": "1.2.0", "_view_name": "StyleView", "description_width": ""}}, "d3aa76e6fc6146289c76da0f67ac752f": {"model_module": "@jupyter-widgets/base", "model_name": "LayoutModel", "state": {"_model_module": "@jupyter-widgets/base", "_model_module_version": "1.2.0", "_model_name": "LayoutModel", "_view_count": null, "_view_module": "@jupyter-widgets/base", "_view_module_version": "1.2.0", "_view_name": "LayoutView", "align_content": null, "align_items": null, "align_self": null, "border": null, "bottom": null, "display": null, "flex": null, "flex_flow": null, "grid_area": null, "grid_auto_columns": null, "grid_auto_flow": null, "grid_auto_rows": null, "grid_column": null, "grid_gap": null, "grid_row": null, "grid_template_areas": null, "grid_template_columns": null, "grid_template_rows": null, "height": null, "justify_content": null, "justify_items": null, "left": null, "margin": null, "max_height": null, "max_width": null, "min_height": null, "min_width": null, "object_fit": null, "object_position": null, "order": null, "overflow": null, "overflow_x": null, "overflow_y": null, "padding": null, "right": null, "top": null, "visibility": null, "width": null}}}

</pre>
</details>

**Comment by vidartf** [comment]
Other than the unwanted print, does everything work as expected, or is there anything that gets broken as well?

**Comment by arghyadeep99** [comment]
Rest everything works fine. Only those unncessary comments are flogging my end of webpage.

**Comment by maartenbreddels** [comment]
I guess we should remove https://github.com/voila-dashboards/voila/blob/112163d88a2d1a5c7706327e68825ddc02819d3a/share/jupyter/voila/templates/default/nbconvert_templates/lab.tpl#L258

**Comment by arghyadeep99** [comment]
I see @maartenbreddels. So, are you making a PR for the same?

**Comment by jtpio** [comment]
xref https://github.com/voila-dashboards/voila/issues/491 as it looks similar.

**Comment by arghyadeep99** [comment]
Yes @jtpio this issue looks similar. So, what do we do about it? :sweat:
